server/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <format>
+#include <iostream>
+
+#include <httplib.h>
+#include <glaze/glaze.hpp>
+#include <vencord/audio.hpp>
+
+template <>
+struct glz::meta<vencord::node>
+{
+    using T                     = vencord::node;
+    static constexpr auto value = object("name", &T::name, "id", &T::id);
+};
+
+struct link_body
+{
+    std::string target;
+};
+
+template <>
+struct glz::meta<link_body>
+{
+    using T                     = link_body;
+    static constexpr auto value = object("target", &T::target);
+};
+
+int main(int argc, char **args)
+{
+    using vencord::audio;
+
+    auto port = 3030;
```

Comment:
maybe it would be better to use a random port like `7591` so that people can just use the default port with very low chances of conflicts